scripts/launch_analysis.sh

```diff
@@ -4,20 +4,36 @@
 # sleep(900); ./mev exec alembic upgrade head; sleep(60); nohup bash scripts/launch_analysis.sh > analysis.out 2>&1 &
 # Input the pool Id of mev-inspect (can also be found on your TILT interface)
 mevInspectPoolId=$(kubectl get pods | sed -n -e '/^mev-inspect-/p' | sed '/^mev-inspect-workers/d' | awk '{print $1}')
+kubectl delete $mevInspectPoolId:resources/*.csv;
 # Input the starting and ending blocks you want to run the profit analysis for
 blockFrom=$((34500000))
-nBlocks=500000
+nBlocks=3000
 blockTo=$((blockFrom+nBlocks))
+window=$((1000))
+reps=$(((${blockTo}-${blockFrom})/${window}))
 startDate=$(date +%s)
 startDateFormatted=$(date +"%Y-%m-%d-%H:%M:%S.")$((startDate))
 echo "Starting analysis from block=${blockFrom} for ${nBlocks} blocks on the ${startDateFormatted}"
-./mev inspect-many $blockFrom $blockTo
-./mev analyze-profit $blockFrom $blockTo True
+echo "${reps}"
+for i in $(seq 0 1 $reps)
```

Comment:
Can we and should we find a way to run these loops in //? @0xpanoramix

Reply:
We can and yes we should, but let's do this in another PR.